EmerisHQ/demeris
# improvements: sort receive assets by marketcap

## Description

- sort receive assets by marketcap
- disallow pay asset in receive

Fixes: #1719 
Fixes: #1741 

## Feature flags

NA

## Testing

1. Select pay asset.
2. Check if pay asset is available in the receive asset selection list and the list is sorted by marketcap.
